src/content/topics/integrations/app-performance/honeycomb/index.mdx

```diff
@@ -9,18 +9,19 @@ published: true
 <CalloutTitle>Honeycomb integrations are Pro and Enterprise features</CalloutTitle>
 <CalloutDescription>
 
-Honeycomb integrations are available to customers on a Pro or Enterprise plan. To learn more, [read about our pricing](https://launchdarkly.com/pricing/). To upgrade your plan, [contact Sales](https://launchdarkly.com/contact-sales/).
+LaunchDarkly’s Pro or Enterprise plan customers can use Honeycomb integrations. To learn more, [read about our pricing](https://launchdarkly.com/pricing/). To upgrade your plan, [contact Sales](https://launchdarkly.com/contact-sales/). You can sign up for a [free Honeycomb account] (https://bit.ly/3qoeonY).
```

Comment:
```suggestion
Honeycomb integrations are available to customers on a Pro or Enterprise plan. To learn more, [read about our pricing](https://launchdarkly.com/pricing/). To upgrade your plan, [contact Sales](https://launchdarkly.com/contact-sales/). 
```
These callouts are uniform across all integrations, and we don't want to differ here.

Reply:
The wording appears to be imply that one needs to be a Honeycomb Pro or Enterprise Plan customer before they use the integration with LaunchDarkly.  This has been brought up a couple of times.  Do you have other alternatives in mind to rectify this issue?

Reply:
@gopalraghavan I understand the concern! The use of the possessive pronoun "our" in the following sentence, as well as the prompt to upgrade your plan by contacting LaunchDarkly's sales organization, both provide context to tell the reader this callout refers to LaunchDarkly's customers, not Honeycomb's. 

I get why this would be unclear, though, so I've updated the callout to include the word "LaunchDarkly."